app/scripts/controllers/startUrl.js

```diff
@@ -87,7 +87,7 @@ angular.module('splain-app')
 
       function setStartUrl () {
         // Using $location object breaks the tests.
-        return window.location.protocol + '//' + window.location.hostname + ':9001/tmdb/_search?stored_fields=*';
+        return window.location.protocol + '//' + window.location.host + '/es_proxy/tmdb/_search?stored_fields=*';
```

Comment:
Do `tmdb` and `stored_fields=*` make sense for any arbitrary ES cluster?

Reply:
I could remove, it'll be up to the user to type in the index name + url.

Reply:
Ideally with this proxy setup, the user would only have to type
```json
/my_index/_search?q=...
```

and it would automatically go through the proxy. heck - if angular is doing an AJAX call, you don't even need to tack on `window.location.protocol + '//' + window.location.host` - it'll get used implicitly.

Reply:
Changed this to return a template URL: user will have to fill in the details.